deploy/examples/create-external-cluster-resources.py

```diff
@@ -556,23 +556,30 @@ def endpoint_dial(self, endpoint_str, timeout=3, cert=None):
         if isinstance(self.cluster, DummyRados):
             return
         protocols = ["http", "https"]
+        response_error = None
         for prefix in protocols:
             try:
                 ep = "{}://{}".format(prefix, endpoint_str)
+                verify = None
                 # If verify is set to a path to a directory,
                 # the directory must have been processed using the c_rehash utility supplied with OpenSSL.
                 if prefix == "https" and cert and self._arg_parser.rgw_skip_tls:
```

Comment:
The `cert` variable is not relevant here I think.
```suggestion
                if prefix == "https" and self._arg_parser.rgw_skip_tls:
```

Reply:
okay I see